SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security
+
+C2PA open source repos are maintained in partnership with Adobe. At this time, security policies are maintained solely by Adobe employees.
```

Comment:
Do we want to list the versions of software that we support? That is sometimes included in security.md files? If it isn't really relevant at this time, we can punt for now.